### ISSUE TITLE
Disable/enable (gdpr)

### ISSUE THREAD

**Issue by andredewaard** [mib200/vue-gtm#20]
Is there some way to disable/enable the tracking of events and views?
For the new rules of GDPR i want a simple button that sets a cookie and will disable/enable tracking without an refresh.

I can write an `if` statement around every call i do with `this.$gtm` but if i can just disable it with something like `this.$gtm.disable()` will be very helpfull.

**Comment by mib200** [comment]
@andre-dw During init, you can do

```
Vue.use(VueGtm, {
  enabled: true, // defaults to true. Plugin can be disabled by setting this to false for Ex: enabled: !!GDPR_Cookie
})
```

I am updating the document is required. Also will add this `disable()` function to do it easily. PR is most welcome :)

**Comment by andredewaard** [comment]
Yes i knew that.
If i can find some time to implement this new feature i will create an PR.

**Comment by mib200** [comment]
closed by latest merge #21
